SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```

Comment:
To make this table more clear for the purpose of my comments below, here's a screenshot:

![Screen Shot 2022-01-03 at 10 24 35 AM](https://user-images.githubusercontent.com/21006/147948492-6eafa8d4-46af-44ba-9061-3e76b47f668f.png)

I don't know if we write this down anywhere but generally we encourage installations to upgrade in order to receive security updates. If a flaw is found in an old version, we're not very likely to put out a point release to fix it.